.github/workflows/ci.yml

```diff
@@ -146,31 +134,19 @@ jobs:
         run: sudo chown -R runner ~/.cargo/registry
       - name: Set grcov version
         run: echo "::set-env name=GRCOV_VERSION::$(cargo search --limit 1 grcov | head -n1 | cut -d '"' -f2)"
-      - name: Cache cargo registry
-        uses: actions/cache@v1
-        with:
-          path: ~/.cargo/registry
-          key: test-registry-${{ hashFiles('**/Cargo.lock') }}
-      - name: Cache target
-        id: cache-target
-        uses: actions/cache@v1
-        with:
-          path: target
-          key: test-target-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('**/Cargo.lock') }}
-      # actions/cache does not cache files, only directories, so we handle the Grcov executable separately.
-      - name: Cache grcov
-        id: cache-grcov
-        uses: actions/cache@v1
-        with:
-          path: ~/grcov
-          key: grcov-${{ env.GRCOV_VERSION }}
+      - name: Cache cargo registry, target, and grcov binary
+        id: test-grcov
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/bin/grcov
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
I think, the cache key should include `grcov` version.

Reply:
Added